src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -523,6 +523,44 @@ public void TestStartWithRootUser()
             RunTestAsSudo(TestStartWithUserName, GetCurrentRealUserName());
         }
 
+        /// <summary>
+        /// Tests when running as root and starting a new process as a normal user,
+        /// the new process doesn't have elevated privileges.
+        /// </summary>
+        [Fact]
+        [OuterLoop("Needs sudo access")]
+        [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
```

Comment:
Can we run this test on the CI machines?

Reply:
Don't know why one above does.

Reply:
I guess if I leave out the OuterLoop attribute that isn't enough to make this test run for the PR?

Reply:
I looked at one or two CI jobs and I don't see `-notrait category=requireselevation`. So I assume they are run as a user that is "sudoable" otherwise we would see tests that require elevation start failing. cc @safern.

@tmds you can request outerloop runs explicitly. eg in the new syntax

/AzurePipelines run corefx-outerloop-linux
/AzurePipelines run corefx-outerloop-windows

Reply:
@safern maybe I didn't do that right?

Reply:
Actually that is correct however comment triggers are broken now for fork PRs.

https://github.com/dotnet/corefx/issues/35121